src/cli.rs

```diff
@@ -39,9 +39,10 @@ pub(crate) struct Args {
     #[clap(
         short = 'p',
         long = "platform",
+        action = ArgAction::Append,
         possible_values = ["linux", "macos", "windows", "sunos", "osx", "android"],
     )]
-    pub platform: Option<PlatformType>,
+    pub platform: Option<Vec<PlatformType>>,
```

Comment:
I think it should now be:
```suggestion
    pub platforms: Option<Vec<PlatformType>>,
```
Can you verify that this change doesn't change the CLI argument to `--platforms` then? I think it should work like this though

Reply:
Good spot yeah I have updated and tested this. It is still `--platform` from the `clap` `long` annotation